### DISCUSSION TITLE
Welcome to Azumi Discussions!

### DISCUSSION THREAD

**Discussion by AquaQuokka** [AquaQuokka/Azumi#1] [Announcements]
Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AquaQuokka** [comment]
Testing

**Comment by TYP3D** [comment]
We could possibly open source with the members who would like to help?

**Reply by AquaQuokka** [reply]
Perhaps, but that's still kinda a form of closed-sourcing. We would only share it with people who we can really trust.

Also, when you say giving the code to members who would like to help, do you mean read-only or being able to edit it?

Because everything you can do by stealing a token, you can do by editing code. While you can reset a token, you can't get rid of information once it is out there. So we'll have to be careful.

But yeah, on the whole, not a bad idea.

**Reply by AquaQuokka** [reply]
Oh also we should probably make a wiki on this repo

**Reply by AquaQuokka** [reply]
I'll make a new discussion and we'll continue talking there.

**Comment by astradotexe** [comment]
🎉
